### ISSUE TITLE
Failed download `yegor256/objectionary/../sprintf.eo`

### ISSUE THREAD

**Issue by i582** [cqfn/eo#509]
I am trying to compile a simple program following the instructions from the readme, after running the command `mvn clean test` I get the following error:

```
[ERROR] Failed to execute goal org.eolang:eo-maven-plugin:0.21.0:assemble (default) on project
 eo-parent: Execution default of goal org.eolang:eo-maven-plugin:0.21.0:assemble failed: 
org.apache.maven.plugin.MojoFailureException: Failed to execute org.eolang.maven.PullMojo:
 Failed while trying to save to ./target/eo/04-pull/org/eolang/sprintf.eo: 
https://raw.githubusercontent.com/yegor256/objectionary/master/objects/org/eolang/sprintf.eo 
-> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

Apparently the file `sprintf.eo` was moved from the root to the `txt` folder and the plugin cannot find it when building.

**Comment by yegor256** [comment]
@i582 you just forgot to do `+alias org.eolang.txt.sprintf` at the top of your program, I believe

**Comment by i582** [comment]
> @i582 you just forgot to do `+alias org.eolang.txt.sprintf` at the top of your program, I believe

Oh, it seems really so, thanks. In the first example in the readme it seems necessary to add a line with alias as well.

**Comment by yegor256** [comment]
@i582 fixed in e56caa50 thanks!
